server/handler.py

```
@@ -1,184 +1,181 @@
-        ):
-            text += self.cli_args.auto_punctuation[0]
-        _LOGGER.debug("Synthesize: raw_text=%r, text=%r", raw_text, text)
-
-        audio = (
-            self.handle_tts_request(text) if text else AudioSegment.silent(duration=0)
-        )
-        await self._output_audio_stream(audio)
-        _LOGGER.debug("Completed legacy request")
-        return True
+        await self.write_event(AudioStop().event())
+        _LOGGER.debug(f"AudioStop sent after processing all batches")
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)